modules/virt-deleting-templates-from-custom-namespace.adoc

```diff
@@ -0,0 +1,51 @@
+// Module included in the following assemblies:
+//
+// * virt/vm_templates/virt-deleting-templates-from-custom-namespace.adoc
+
+[id="virt-deleting-templates-from-custom-namespace_{context}"]
+= Deleting templates from a custom namespace
+
+When you delete Red Hat templates that are deployed to a custom namespace, they are regenerated. To avoid regenerating the templates, replace the existing value for the custom namespace with a different value. This causes the templates to be deployed to the custom namespace with the new value. You can now delete the templates from the original custom namespace.
+
+.Procedure
+
+. Open the `hco` CR to edit the file.
++
+[source,terminal]
+----
+$ oc edit hco
+----
++
+. Modify the value for the `commonTemplateNamespace` attribute by replacing it with a new value for the custom namespace.
```

Comment:
I think we should not use a 'new value' here because it deploys templates in the new namespace, which causes unexpected results. It should either change back to the previous/default value or remove `commonTemplateNamespace` from the spec.

Reply:
Thanks for the note, @gouyang. Requesting @ksimon1 for his input as well because this content was developed based on the steps that he demonstrated.

Reply:
@gouyang - Requesting clarification so that I understand the requested change. Can you tell me what is the change that needs to be made in the documentation here? Can you please provide me with specific instructions on what changes need to be made to this step?

Reply:
The current instructions guides users to deploy templates to a new namespace like `newdefaultnamespace` to make it able to delete templates in current namespace, what `newdefaultnamespace` should be? Isn't it deploying templates to another namespace? 

Hence, I suggested:
1. Just use the CNV default template's namespace `openshift`.
2. If we don't want bind it with `openshift`, just remove this line from the spec.

Example:
```
. Modify the value for the `commonTemplateNamespace` attribute by replacing it with the openshift template's default namespace 'openshift'.
```
Or
```
. Remove `commonTemplateNamespace` from the spec.
```
And modify step 3 accordingly.

Reply:
@gouyang - When the user attempts to delete templates from the custom namespace, those templates get regenerated automatically. Therefore, @ksimon1 provided the steps for this procedure where the user can avoid having those templates regenerated after deletion. Are you stating that we should not follow the process established by @ksimon1. If so, what is the alternative procedure that prevents the templates from being regenerated when they are deleted? Requesting clarification.

Reply:
@gouyang @lmandavi The admin can alter the custom template namespace at any time, regardless of modifying or removing it. However, I do think @gouyang's point of keeping the operations to add/remove will be the most clear.

Reply:
My point is don't deploy templates to another namespace to make it able to delete template in current namespace.

Reply:
@gouyang Can you clarify? I'm not sure I understand. The purpose for the feature is allowing the admin to decide which templates are available to the user, they will delete those they don't want available from the default namespace.

Reply:
@gouyang, @sarahbx - Because of continuing discussions  and requests for clarifications for this procedure, I am requesting either of you to email me or Gchat with a complete procedure with numbered steps and command syntax for each step. This will ensure that we can get the fastest resolution and closure on this item. When you provide your procedure with steps, I can remove the procedure with steps provided by @ksimon1 and replace it with your procedure. 
Adding @ksimon1 for awareness.

Reply:
Discussed with @lmandavi who will schedule follow-up with @gouyang and myself. Will simplify to `default -> custom -> default` use cases.

Reply:
Updated the content. User is instructed to remove the `commonTemplatesNamespace` from the spec.